CHANGES.rst

```diff
@@ -5,6 +5,7 @@ Version 0.9.0 (UNRELEASED)
 ---------------------------
 
 - Adds Launch on REANA page allowing the submission of workflows via badge-clicking.
+- Adds redirection to the page requested by the user after logging-in using SSO.
```

Comment:
Since this is basically a bug fix, I would suggest something like:

- Fixes redirection chain for non-signed-in CERN SSO users to access the desired target page after sign-in.